src/components/Profile.tsx

```diff
@@ -9,9 +9,12 @@ import { saveProfile } from "../services/sdk";
 import { core } from "@dsnp/sdk";
 
 const Profile = (): JSX.Element => {
+  const myId: DSNPUserId | undefined = useAppSelector((state) => state.user.id);
```

Comment:
It'd be better to keep this one as `userId` an make the other `displayId`, `idToDisplay` or some such, since we use `userId` throughout the app to mean the current logged in user.